lingam/lina.py

```diff
@@ -0,0 +1,629 @@
+"""
+Python implementation of the LiNGAM algorithms.
+The LiNGAM Project: https://sites.google.com/site/sshimizu06/lingam
+"""
+
+import numpy as np
+import scipy.optimize as sopt
+from scipy.special import expit as sigmoid
+from factor_analyzer import (ConfirmatoryFactorAnalyzer, ModelSpecificationParser)
+import random
```

Comment:
'random' imported but unused

Reply:
Thanks for your carefulness! Please delete "import random".